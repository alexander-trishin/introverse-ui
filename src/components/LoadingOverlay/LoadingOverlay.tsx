import './LoadingOverlay.scss';

const LoadingOverlay = () => {
    return (
        <div className="loading-overlay">
            <div className="loader">
                <div>
                    <div className="arrow up outer-18"></div>
                    <div className="arrow down outer-17"></div>
                    <div className="arrow up outer-16"></div>
                    <div className="arrow down outer-15"></div>
                    <div className="arrow up outer-14"></div>
                </div>
                <div>
                    <div className="arrow up outer-1"></div>
                    <div className="arrow down outer-2"></div>
                    <div className="arrow up inner-6"></div>
                    <div className="arrow down inner-5"></div>
                    <div className="arrow up inner-4"></div>
                    <div className="arrow down outer-13"></div>
                    <div className="arrow up outer-12"></div>
                </div>
                <div>
                    <div className="arrow down outer-3"></div>
                    <div className="arrow up outer-4"></div>
                    <div className="arrow down inner-1"></div>
                    <div className="arrow up inner-2"></div>
                    <div className="arrow down inner-3"></div>
                    <div className="arrow up outer-11"></div>
                    <div className="arrow down outer-10"></div>
                </div>
                <div>
                    <div className="arrow down outer-5"></div>
                    <div className="arrow up outer-6"></div>
                    <div className="arrow down outer-7"></div>
                    <div className="arrow up outer-8"></div>
                    <div className="arrow down outer-9"></div>
                </div>
            </div>
        </div>
    );
};

export default LoadingOverlay;
