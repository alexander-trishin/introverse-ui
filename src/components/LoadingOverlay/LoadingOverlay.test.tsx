import { render } from '@testing-library/react';

import LoadingOverlay from './LoadingOverlay';

describe('<LoadingOverlay />', () => {
    it('should match snapshot', () => {
        const { container } = render(<LoadingOverlay />);

        expect(container).toMatchSnapshot();
    });
});
