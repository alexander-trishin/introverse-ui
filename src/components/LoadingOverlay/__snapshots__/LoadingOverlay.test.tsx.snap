// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<LoadingOverlay /> should match snapshot 1`] = `
<div>
  <div
    class="loading-overlay"
  >
    <div
      class="loader"
    >
      <div>
        <div
          class="arrow up outer-18"
        />
        <div
          class="arrow down outer-17"
        />
        <div
          class="arrow up outer-16"
        />
        <div
          class="arrow down outer-15"
        />
        <div
          class="arrow up outer-14"
        />
      </div>
      <div>
        <div
          class="arrow up outer-1"
        />
        <div
          class="arrow down outer-2"
        />
        <div
          class="arrow up inner-6"
        />
        <div
          class="arrow down inner-5"
        />
        <div
          class="arrow up inner-4"
        />
        <div
          class="arrow down outer-13"
        />
        <div
          class="arrow up outer-12"
        />
      </div>
      <div>
        <div
          class="arrow down outer-3"
        />
        <div
          class="arrow up outer-4"
        />
        <div
          class="arrow down inner-1"
        />
        <div
          class="arrow up inner-2"
        />
        <div
          class="arrow down inner-3"
        />
        <div
          class="arrow up outer-11"
        />
        <div
          class="arrow down outer-10"
        />
      </div>
      <div>
        <div
          class="arrow down outer-5"
        />
        <div
          class="arrow up outer-6"
        />
        <div
          class="arrow down outer-7"
        />
        <div
          class="arrow up outer-8"
        />
        <div
          class="arrow down outer-9"
        />
      </div>
    </div>
  </div>
</div>
`;
